tools/vf-component-library/src/site/_includes/layouts/components.njk

```diff
@@ -8,7 +8,7 @@ templateEngineOverride: njk
 {% set variants = component.variants.items %}
 
 <div class="embl-grid">
-  <nav class="vf-tree vf-search-client-side--no-index">
+  <nav class="vf-tree vf-search-client-side--no-index" style="z-index: 5150;">
```

Comment:
I still want to update the layout _somehow_ so that things like fullbleed preview correctly ...

Reply:
I added this - because without it - it's hard to click on things in the sidebar, especially when you're on the `vf-hero` component.

Reply:
Are you happy with this, so I can resolve this?

Reply:
Yeah, these were just comments